apps/www/src/element/ssz-shacl-report.js

```diff
@@ -0,0 +1,86 @@
+import { html, LitElement } from 'lit'
+import TermMap from '@rdfjs/term-map'
+import { sh } from '@tpluscode/rdf-ns-builders'
+import '@shoelace-style/shoelace/dist/components/details/details.js'
+import { toSparql } from 'clownface-shacl-path'
+
+customElements.define('ssz-shacl-report', class extends LitElement {
```

Comment:
This is the dialog element which displays shacl reports

Reply:
This is probably a good candidate to be in a library somewhere. It's quite common to show validation reports.

I'm not sure which lib though

Reply:
There is no such lib right now. Could put it in the https://github.com/zazuko/rdfjs-elements/ repo (even if a new package)

Will create an issue

Reply:
That would be awesome; I've also seen other apps that don't know how to show such a report. It's nice to have it centralized